serving/docker/pytorch-gpu.Dockerfile

```diff
@@ -21,6 +21,8 @@ ARG torch_vision_version=0.19.0
 ARG onnx_version=1.19.0
 ARG python_version=3.10
 ARG numpy_version=1.26.4
+ARG pydantic_version=2.8.2
+ARG djl_converter_wheel="https://publish.djl.ai/djl_converter/djl_converter-0.31.0-py3-none-any.whl"
```

Comment:
Curious, What is this for?

Reply:
1. `djl-converter` is used to convert huggingface model automatically to OnnxRuntime/Rust on the fly to optimize the performance
2. we depends on peft and pydantic when using `huggingface.py`, we should consider move them to where it used. right now they are hard dependency if run any python model even not related to peft in the container.